### ISSUE TITLE
Could you let me know, each output in dataloader?

### ISSUE THREAD

**Issue by LeeDoYup** [asdf0982/vqa-mfb.pytorch#9]
I feel hard to understand what is the meaning (and shape) of each outputs in Dataloader.
```
 return word, word_length, feature, answer, glove_matrix, epoch
```
Could you explain?

**Comment by asdf0982** [comment]
@LeeDoYup These names are really not easy to understand, I also recalled for a while...
- word: a list of words which are the tokens in the question. like[12, 345, 678, ..., 90]
- word_length: number of words in the question
- feature: image feature extracted by CNN
- answer: 3000 dimensions ground truth answer label. like [0,0,0,0,1,0,...,0], but not all one-hot embed
- glove_matrix: question word embed using spacy glove
- epoch: current epoch

**Comment by LeeDoYup** [comment]
Then, the **shape** of 
```
- feature: [# of object, feature dimensionality] 
- glove_matrix: [vocabulary set size, embedding size]
```
right? Thx for your kind reply !!

**Comment by asdf0982** [comment]
@LeeDoYup glove_matrix is [max length of sentence, glove emedding size]
Such as [15, 300], the sentence has a maximum of 15 words, and each word-token is mapped to 300 dimensions.

**Comment by LeeDoYup** [comment]
Really Thx :)
